packages/web/src/pages/feed-page/components/FollowUsers.tsx

```diff
@@ -0,0 +1,82 @@
+import { useCallback } from 'react'
+
+import { useGetFeaturedArtists } from '@audius/common/api'
+import { feedPageActions } from '@audius/common/store'
+import { Button, Flex, IconUserFollow as IconFollow } from '@audius/harmony'
+import { Form, Formik } from 'formik'
+import { useDispatch } from 'react-redux'
+
+import { FollowArtistCard } from 'components/follow-artist-card/FollowArtistCard'
+import { SelectArtistsPreviewContextProvider } from 'components/follow-artist-card/selectArtistsPreviewContext'
+
+import styles from './FollowUsers.module.css'
+
+const messages = {
+  cta: `Let’s fix that by following some of these artists!`,
+  noFollowers: `Oops! There's nothing here.`
+}
+
+type FollowUsersValues = {
+  selectedArtists: string[]
+}
+
+const initialValues: FollowUsersValues = {
+  selectedArtists: []
+}
+
+const FollowUsers = () => {
+  const dispatch = useDispatch()
+
+  const { data: featuredArtists } = useGetFeaturedArtists(undefined)
+
+  const handleSubmit = useCallback(
+    (values: FollowUsersValues) => {
+      const { selectedArtists } = values
+      const followUsers = selectedArtists.map((id) => parseInt(id))
+      dispatch(feedPageActions.followUsers(followUsers))
+    },
+    [dispatch]
+  )
+
+  return (
+    <SelectArtistsPreviewContextProvider>
+      <Formik initialValues={initialValues} onSubmit={handleSubmit}>
+        {({ values }) => (
+          <Form>
+            <div className={styles.followText}>
+              <div>
+                {messages.noFollowers}{' '}
+                <i className='emoji face-screaming-in-fear' />
+              </div>
+              <div>{messages.cta}</div>
+            </div>
+            <Flex
+              inline
+              w='100%'
+              justifyContent='center'
+              wrap='wrap'
+              gap='m'
+              m='s'
+            >
+              {featuredArtists?.map((artist) => (
+                <FollowArtistCard key={artist.user_id} user={artist} />
+              ))}
```

Comment:
do we want to do anything fancy to make sure we never get a row of just 1 artist at the bottom?

Reply:
Naw I think fine for now. We eventually will improve this with explore updates